houston/model/build.js

```diff
@@ -145,12 +145,7 @@ schema.methods.doBuild = function () {
     tag: this.cycle.tag,
     version: this.cycle.version
   })
-  .then(() => {
-    log.debug(`Building ${this.cycle.name} for ${this.arch} on ${this.dist}`)
```

Comment:
I removed any log functions from all the models, but if you think it's helpful I can add it back.